packages/@webex/internal-plugin-metrics/package.json

```diff
@@ -37,7 +37,7 @@
   "dependencies": {
     "@webex/common": "workspace:*",
     "@webex/common-timers": "workspace:*",
-    "@webex/event-dictionary-ts": "^1.0.1819",
+    "@webex/event-dictionary-ts": "^1.0.1930",
```

Comment:
Can you upgrade this package in webex-js-sdk/packages/@webex/plugin-meetings/package.json as well please?